ui/store/actions.ts

```diff
@@ -3368,6 +3368,19 @@ export function dismissOpenSeaToBlockaidBanner(): ThunkAction<
   };
 }
 
+export function setRedesignedConfirmationsEnabledFeature(
```

Comment:
As mentioned in another confirmation, we can re-use `setPreference` if we include the new boolean inside the `preferences` object.

Reply:
done